homeassistant/components/sensor/darksky.py

```diff
@@ -117,6 +118,8 @@
     vol.Required(CONF_API_KEY): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_UNITS): vol.In(['auto', 'si', 'us', 'ca', 'uk', 'uk2']),
+    vol.Optional(CONF_LATITUDE): cv.latitude,
```

Comment:
Please ensure that latitude and longitude are always set in a group like here: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/influxdb.py#L35-L36